### DISCUSSION TITLE
Syntax for using custom properties in templates

### DISCUSSION THREAD

**Discussion by rsandidfc** [bsorrentino/maven-confluence-plugin#272] [Q&A] [answered]
Hi folks - what is the syntax for injecting custom properties in the template.confluence file? In the sample poms given, there are example properties:

```
<properties>
	<prop>the_text_value</prop><!-- SIMPLE TEXT -->
	<prop1>classpath:plugin-report.properties</prop1><!-- FROM CLASSPATH -->
	<prop2>file://${basedir}/confluence.html</prop2><!-- FROM FILE SYSTEM -->
</properties>
```

But I don't see anywhere that the above are actually used.

Thanks!

**Comment by bsorrentino** [comment, marked as answer]
HI @rsandidfc 

Custom properties are injected in every document template and their value could be used through the variable template: 

> `${`variable name`}`

In your sample

```xml
<properties>
	<prop>the_text_value</prop><!-- SIMPLE TEXT -->
	<prop1>classpath:plugin-report.properties</prop1><!-- FROM CLASSPATH -->
	<prop2>file://${basedir}/confluence.html</prop2><!-- FROM FILE SYSTEM -->
</properties>
```

the equivalent properties are :

```
* ${prop}
* ${prop1}
* ${prop2}

```
